packages/api/src/entities/Destination.ts

```diff
@@ -0,0 +1,37 @@
+/* istanbul ignore file */
+import { Entity, ManyToOne, Property } from '@mikro-orm/core';
+import { ConstructorValues } from '../utils/types';
+import { Node } from './Node';
+import { Code } from './Code';
+
+export type DestinationConstructorValues = ConstructorValues<Destination>;
+
+@Entity()
+export class Destination extends Node<Destination> {
+  @Property({ columnType: 'text' })
+  origin: string;
+
+  @Property()
```

Comment:
typically we want to specify the data type for all the columns we create, in this case it would probably be an `integer`
```suggestion
  @Property({ columnType: 'int' })
```